site/.dumirc.ts

```diff
@@ -189,6 +197,14 @@ export default defineConfig({
             },
             order: 9,
         },
+        {
+          slug: 'api/ntv',
+          title: {
+              zh: 'NarrativeTextVis（NTV）',
```

Comment:
zh is English and use Chinese brackets?

Reply:
`NarrativeTextVis` is react component name. 🤔

Reply:
Changed to English brackets.